config/prometheus/collector_alerts.yaml

```diff
@@ -55,6 +55,36 @@ spec:
       labels:
         service: collector
         severity: Warning
+    - alert: ElasticsearchDeprecation
+      annotations:
+        message: "The OpenShift Elasticsearch Operator is deprecated and is planned to be removed in a future release. Red Hat provides bug fixes and support for this feature during the current release lifecycle, but this feature no longer receives enhancements. As an alternative to using the OpenShift Elasticsearch Operator to manage the default log storage, you can use the Loki Operator."
+        summary: "Detected Elasticsearch as the on cluster storage which is being deprecated in a future release."
```

Comment:
Aren't all these components already deprecated? I think we're better suited with "going to be removed in a future release".

edit: it seems Jeff had a similar thought.